packages/neuron-ui/src/components/GeneralSetting/LockWindowDialog/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useEffect, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import Dialog from 'widgets/Dialog'
+import SplitPasswordInput from 'widgets/SplitPasswordInput'
+import Alert from 'widgets/Alert'
+import Button from 'widgets/Button'
+import { passwordLength, useOldPassword, usePassword, useRepeatPassword } from './hooks'
+import styles from './lockWindowDialog.module.scss'
+
+const LockWindowDialog = ({
```

Comment:
i'd suggest renaming the `LockWindowDialog` to `LockWindowUnlockDialog` for more clarity to avoid confusion with the `LockWindow`

Reply:
This dialog is used for setting a password to lock the window or change the password. It is not used to unlock the window.